stable/bookstack/values.yaml

```diff
@@ -160,3 +160,13 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+## Enable ldap authentication. See https://www.bookstackapp.com/docs/admin/ldap-auth/ for details on how to set it up.
+ldap:
+  enabled: true
+  server: false
+  base_dn: false
+  dn: false
+  pass: false
+  userFilter: false
+  version: false
```

Comment:
how is `false` a valid input for `version` ?

Reply:
While it does look weird, I copied those values from defaults in the docker image the chart uses. In this case, `false` represents the defaults of the image. Please see docker image repo: https://github.com/solidnerd/docker-bookstack/blob/master/docker-entrypoint.sh

Reply:
I still think it looks confusing.  Can we default them to empty and let the bootstrap default to false because they're empty ?

Reply:
I agree, these nonsensical defaults should not make it into the chart.

Reply:
I replaced all but `enabled` with empty values